mirrord-schema.json

```diff
@@ -790,7 +790,7 @@
         },
         "readlink": {
           "title": "_experimental_ readlink {#experimental-readlink}",
-          "description": "Enables the `readlink` hook.",
+          "description": "DEPRECATED, WILL BE REMOVED",
```

Comment:
This is for ipv6 only right?

Reply:
no, I just nitpicked a fix (readlink was deprecated but docs stayed same which was weird)